NEWS.md

```diff
@@ -221,7 +221,7 @@
     # [1] "A" "B" "C"
     setnames(DT, 2:3, tolower)
     names(DT)
-    # [1] "A" "b" "c"
+    # [1] "a" "b" "c"
```

Comment:
@jangorecki Why this change?   The `setnames` call sets `2:3` so `c("A","b","c")` was correct.

Reply:
I guess the 2:3 wasn't clear enough when reading quickly.  I changed it to c(1,3) to make it clearer in 7cad29b

Reply:
Oh, yes, I haven't notice 2:3! Thanks